infrastructure/nomad/playbooks/templates/jobs/mev-commit-oracle.nomad.j2

```diff
@@ -42,6 +42,15 @@ job "{{ job.name }}" {
       port = "{{ port_name }}"
       tags = ["{{ port_name }}"]
       provider = "nomad"
+      {% if port_name == "http" %}
+      check {
+        name     = "HTTP Health Check"
```

Comment:
The name doesn't make much sense when there is no service name in it, so I would suggest omitting it. The same is below.